platform/core/src/log.js

```diff
@@ -4,8 +4,23 @@ const log = {
   info: console.log,
   trace: console.trace,
   debug: console.debug,
-  time: console.time,
-  timeEnd: console.timeEnd,
+  time: key => {
+    log.timingKeys[key] = true;
+    console.time(key);
+  },
+  timeEnd: key => {
+    if (!log.timingKeys[key]) {
+      return;
+    }
+    log.timingKeys[key] = false;
+    console.timeEnd(key);
+  },
+  // Store the timing keys to allow knowing whether or not to log events
+  timingKeys: {
+    // script time values are added before log is loaded, and the log
+    // shouldn't depend on the enums, so for this case recreate the string.
+    scriptToView: true,
+  },
```

Comment:
it would be great if we can have an idea of when does script starts timing and ends